### PR TITLE
Move subroutine hapsequencer.run_sourcelist_comparision() to svm_quality_analysis.py.

### PR DESCRIPTION
**Pull request by dulude**
**Notes**
- [See JIRA ticket HLA-326](https://jira.stsci.edu/browse/HLA-326)

**TL;DR**
- Execution of compare_sourcelists.py has been moved from hapsequencer.py to svm_quality_analysis.py in order to centralize the execution of all quality checks

**Summary**
- moved subroutine hapsequencer.run_sourcelist_comparision() to svm_quality_analysis.py.
- renamed subroutine svm_quality_analysis.run_sourcelist_comparision() to svm_quality_analysis.run_hla_sourcelist_ comparison()
- moved subroutine hapsequencer.correct_hla_classic_ra_dec() to svm_quality_analysis.py.
- added necessary import statements 
- updated impacted subroutine calls.
- updated svm_quality_analysis.py to facilitate execution of run_hla_sourcelist_comparision() from either the command line or as a python subroutine call.
- compare_sourcelists.py: updated subroutine comapresourcelists() to utilize new subroutine inputs json_timestamp and json_time_since_epoch to instantiate the HapDiagnostic objects with the same universal generation time, generation date and time_since_epoch values used elsewhere by all the other SVM QA tests
- compare_sourcelists.py: long (> 80 characters) catalog filenames on statistics pages of pdf file now wrap to the next line so they don't run off of the edge of the page.

### CODE REVIEW

File: drizzlepac/devutils/comparison_tools/compare_sourcelists.py [hunk 1]
```diff
@@ -356,7 +356,15 @@ def computeFlagStats(matchedRA, max_diff, plotGen, plot_title, plotfile_prefix,
                 else:
                     stat_text_blob += "\n"
             stat_text_blob += "\n" + timestamp + "\n"
-            stat_text_blob += "Comparison Sourcelist: {}\nReference Sourcelist:  {}".format(catalog_names[1], catalog_names[0])
+            if len(catalog_names[1]) > 80:
+                out_compcat_filename = "{}\n{}{}".format(catalog_names[1][:80], " "*23, catalog_names[1][80:])
+            else:
+                out_compcat_filename = catalog_names[1]
```
Comment (stsci-hack) [inline drizzlepac/devutils/comparison_tools/compare_sourcelists.py:362]: This set of code is repeated (with different inputs) 4 times, making it suitable for defining a function to to reformat this string.  Something for future PRs....

Comment (dulude) [reply drizzlepac/devutils/comparison_tools/compare_sourcelists.py]: Issue resolved in commit [d4d2042](https://github.com/spacetelescope/drizzlepac/pull/660/commits/d4d204239c1aac6fa2d288df9a216f8f55b2226d)
